solutions/devsprint-vinicius-carvalho-3/DeliveryApp/Screens/AddressSearch/AddressListView.swift

```diff
@@ -8,8 +8,105 @@
 import UIKit
 
 class AddressListView: UIView {
-
-    func updateView(with addresses: [Address]) {
+    
+    private let addressCellIdentifier = "AddressCellIdentifier"
+    
+    private var addresses: [Address] = []
+    
+    private lazy var searchbar: UISearchBar = {
```

Comment:
[Comment]
Talk with Kalina, about this point, asking for her about your task. If needed create another search or not.